.travis.yml

```diff
@@ -7,6 +7,44 @@ dist: bionic
 
 matrix:
   include:
+  # Go 1.15
```

Comment:
I wasn't able to get matrix expansion working here, but figured it's pretty important to avoid silent regressions on Go 1.15 builds.

Was hoping we could simply add:
```
go:
- 1.14.x
- 1.15.x
```

and remove the references to `1.14.x` in the `matrix.include` list, but it seems `go` can't be expanded into the matrix. 

Suggestions welcome!

Reply:
Since the build now passes for 1.15, I suggest we just update 1.14 -> 1.15, instead of duplicating (the CI already runs for too long, we don't have capacity to double the number of steps).

We may want to add one extra step to run unit tests & lint with go tip (but I would do it in a separate PR).

Reply:
Okay, updated to 1.15. 
Where can I learn more about "go tip"?

Reply:
I've seen it used elsewhere, e.g. https://github.com/opentracing/opentracing-go/blob/d34af3eaa63c4d08ab54863a4bdd0daa45212e12/.travis.yml#L7

Reply:
Would an additional "tip" build version result in doubling the number of steps (1.15 + tip)?